mobile-widgets/qmlmanager.cpp

```diff
@@ -1583,6 +1583,18 @@ void QMLManager::setLibdcLog(bool value)
 	emit libdcLogChanged();
 }
 
+bool QMLManager::developer() const
+{
+	return m_developer;
+}
+
+void QMLManager::setDeveloper(bool value)
+{
+	m_developer = value;
+	DCDeviceData::instance()->setSaveLog(value);
+	emit developerChanged();
+}
+
 bool QMLManager::btEnabled() const
```

Comment:
why is this something that's shared with the C++ code? Do we need that information there? I would have thought this was purely a UI issue and could stay in QML...